src/sentry/interfaces/stacktrace.py

```diff
@@ -218,6 +219,8 @@ def slim_frame_data(frames, frame_allowance=settings.SENTRY_MAX_STACKTRACE_FRAME
         else:
             system_frames.append(frame)
 
+    del frames[hard_frame_limit - 1:-1]
```

Comment:
`:-1` is incorrect I think. It leaves the last frame in place. Perhaps that is intended though?

Reply:
That was intentional though not sure if great